zmon_cli/cmds/grafana.py

```diff
@@ -30,6 +30,20 @@ def grafana_get(obj, dashboard_id, output, pretty):
         act.echo(dashboard)
 
 
+@grafana.command('delete')
+@click.argument('dashboard_id', type=click.STRING)
+@click.pass_obj
+@yaml_output_option
+@pretty_json
+def grafana_delete(obj, dashboard_id, output, pretty):
```

Comment:
why do we need this command? it's one more command/API for us to support, whereas we want users to move to Grafana API